pkgs/build-support/rust/test/import-cargo-lock/git-dependency-workspace-inheritance/crate.toml

```diff
@@ -11,3 +11,6 @@ keywords = [
 [dependencies]
 foo = { workspace = true, features = ["cat"] }
 bar = "1.0.0"
+
+[lints]
+workspace = true
```

Comment:
Does it make sense to test this both in `gitDependencyWorkspaceInheritance` and the new `workspaceLints` test?

Reply:
`gitDependencyWorkspaceInheritance` is a catch-all test for all inheritance, it feels nice to have `lints` in it, so everything inherited can be seen in one test. The latter test is an indicating test of the issue described in the PR description, it definitely can't be removed.